src/benchmarklib/tpch/tpch_benchmark_item_runner.cpp

```diff
@@ -297,7 +297,7 @@ std::string TPCHBenchmarkItemRunner::_build_query(const BenchmarkItemID item_id)
     }
 
     case 14 - 1: {
-      std::uniform_int_distribution<> date_diff_dist{0, 47};
+      std::uniform_int_distribution<> date_diff_dist{0, 5*12};
```

Comment:
DATE is the first day of a month randomly selected from a random year within [1993 .. 1997].